### PR TITLE
HAL Schema

### PR DESCRIPTION
**Pull request by whiskeysierra**

### CODE REVIEW

**Comment by tkrop** [comment]
:+1:

File: hyper-media/Hypermedia.md [hunk 1]
```diff
@@ -31,8 +31,32 @@ Links to other resources must be defined exclusively using [HAL](http://stateles
 preferably using standard [link relations](http://www.iana.org/assignments/link-relations/link-relations.xml).
 
 Clients and Servers are required to support `_links` with its `href` and `rel` attributes, not only at the root level
-but also in nested objects. To reduce the effort needed by clients to process hypertext data from servers it's not recommended to serve data with CURIEs, URI templates or embedded resources. Nor is it required to support the HAL media type
-`application/hal+json`.
+but also in nested objects. To reduce the effort needed by clients to process hypertext data from servers it's not recommended to serve data with CURIEs, URI templates or embedded resources. Nor is it required to support the HAL media type `application/hal+json`. The following snippet specifies the HAL link structure in JSONSchema:
```
Comment (fmueller) [inline hyper-media/Hypermedia.md]: I'm not sure if we should exclude embedding resources here as we discuss embedding resources in other sections as well.

Comment (whiskeysierra) [inline hyper-media/Hypermedia.md]: I'd exclude them for now until we have a better understanding how to do it.

Comment (fmueller) [inline hyper-media/Hypermedia.md]: Then we should also express this that we are waiting for more experience on embedding. Additionally we should rework this rule, http://zalando.github.io/restful-api-guidelines/hyper-media/Hypermedia.html#should-allow-optional-embedding-of-subresources

**Comment by fmueller** [comment]
👍

**Pull request merged**